lib/public/components/Filters/common/filters/TextTokensFilterModel.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import { Observable } from '/js/src/index.js';
+
+/**
+ * Model for text input
```

Comment:
This model does a lot more than that. Please try to think from the reader point of view and explain it better!

Reply:
more descriptive docs added